packages/caliper-samples/benchmark/simple/config-ethereum.yaml

```diff
@@ -0,0 +1,53 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The benchmark config files are DLT-agnostic, so you don't need a specific one for ethereum.